extras/init.d/celeryd.default.dist.j2

```diff
@@ -96,13 +96,13 @@ CELERYD_FLOWER="$CELERYD_CHDIR/manage.py celery flower"
 ADMIN_USER="flower_user"
 ADMIN_PASS="flower_pass"
 #Possibly not necessary?
-FLOWER_BROKER_OPT="--broker=amqp://broker_user:broker_pass@hostname:5672/virtualhost_name"
+#FLOWER_BROKER_OPT="--broker=amqp://broker_user:broker_pass@hostname:5672/virtualhost_name"
 #TODO:Change to Google+OAUTH2 VERY soon!
 FLOWER_AUTH_OPT="--basic_auth=$ADMIN_USER:$ADMIN_PASS"
 OAUTH2_ACCESS_LIST=""
 
 FLOWER_LOG_FILE="/var/log/celery/flower.log"
-CELERY_FLOWER_OPTS="--certfile=$SSH_CERT_FILE --keyfile=$SSH_KEY_FILE --port=8443 --log_file_prefix=$FLOWER_LOG_FILE --logging=warn --url_prefix=flower $FLOWER_BROKER_OPT $FLOWER_AUTH_OPT"
+CELERY_FLOWER_OPTS="--certfile=$SSH_CERT_FILE --keyfile=$SSH_KEY_FILE --port=8443 --log_file_prefix=$FLOWER_LOG_FILE --logging=warn --url_prefix=flower $FLOWER_AUTH_OPT"
```

Comment:
Flower can actually determine its BROKER_OPT using just the 'app.settings' that it loads as part of the celery bootstrap process. This was "An Old way" of doing things.